testsuite/single_node_performance.py

```diff
@@ -10,40 +10,73 @@
+    RunGroupConfig(expected_tps=18500, key=RunGroupKey("account-generation"), included_in=Flow.LAND_BLOCKING),
+    RunGroupConfig(expected_tps=3550, key=RunGroupKey("modify-global-resource"), included_in=Flow.LAND_BLOCKING | Flow.REPRESENTATIVE),
+    RunGroupConfig(expected_tps=10600, key=RunGroupKey("modify-global-resource", module_working_set_size=10), included_in=Flow.LAND_BLOCKING),
+    RunGroupConfig(expected_tps=125, key=RunGroupKey("publish-package"), included_in=Flow.LAND_BLOCKING | Flow.REPRESENTATIVE),
+    RunGroupConfig(expected_tps=1600, key=RunGroupKey(
+        "mix_publish_transfer",
+        transaction_type_override="publish-package coin-transfer",
+        transaction_weights_override="1 500",
+    ), included_in=Flow.LAND_BLOCKING, waived=True),
+    RunGroupConfig(expected_tps=390, key=RunGroupKey("batch100-transfer"), included_in=Flow.LAND_BLOCKING),
+    RunGroupConfig(expected_tps=940, key=RunGroupKey("batch100-transfer", executor_type="native"), included_in=Flow.LAND_BLOCKING),
+
+    RunGroupConfig(expected_tps=1550, key=RunGroupKey("token-v1ft-mint-and-transfer"), included_in=Flow.LAND_BLOCKING),
+    RunGroupConfig(expected_tps=7700, key=RunGroupKey("token-v1ft-mint-and-transfer", module_working_set_size=20), included_in=Flow.LAND_BLOCKING),
+    RunGroupConfig(expected_tps=1000, key=RunGroupKey("token-v1nft-mint-and-transfer-sequential"), included_in=Flow.CONTINUOUS),
+    RunGroupConfig(expected_tps=5150, key=RunGroupKey("token-v1nft-mint-and-transfer-sequential", module_working_set_size=20), included_in=Flow.CONTINUOUS),
+    RunGroupConfig(expected_tps=1300, key=RunGroupKey("token-v1nft-mint-and-transfer-parallel"), included_in=Flow.CONTINUOUS),
+    RunGroupConfig(expected_tps=5300, key=RunGroupKey("token-v1nft-mint-and-transfer-parallel", module_working_set_size=20), included_in=Flow.CONTINUOUS),
+
+    # RunGroupConfig(expected_tps=1000, key=RunGroupKey("token-v1ft-mint-and-store"), included_in=Flow.NONE),
+    # RunGroupConfig(expected_tps=1000, key=RunGroupKey("token-v1nft-mint-and-store-sequential"), included_in=Flow.NONE),
+    # RunGroupConfig(expected_tps=1000, key=RunGroupKey("token-v1nft-mint-and-transfer-parallel"), included_in=Flow.NONE),
+
+    RunGroupConfig(expected_tps=18000, key=RunGroupKey("no-op2-signers"), included_in=Flow.CONTINUOUS),
+    RunGroupConfig(expected_tps=18000, key=RunGroupKey("no-op5-signers"), included_in=Flow.CONTINUOUS),
+   
+    RunGroupConfig(expected_tps=1500, key=RunGroupKey("token-v2-ambassador-mint"), included_in=Flow.LAND_BLOCKING | Flow.REPRESENTATIVE),
+    RunGroupConfig(expected_tps=5500, key=RunGroupKey("token-v2-ambassador-mint", module_working_set_size=20), included_in=Flow.LAND_BLOCKING | Flow.REPRESENTATIVE),
+
+    RunGroupConfig(expected_tps=50000, key=RunGroupKey("coin-transfer", executor_type="sharded", sharding_traffic_flags="--connected-tx-grps 5000"), included_in=Flow.REPRESENTATIVE),
```

Comment:
@sitalkedia  this is sharded execution. 

not run on PR or continuous, only when using FLOW=REPRESENTATIVE